### DISCUSSION TITLE
Github Discussion 支持 RSS 订阅了

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#59] [1.General]
可能是个冷知识。我看到这个帖子 https://github.com/orgs/community/discussions/31#discussioncomment-4048867 发现 2022 年 11 月起， Github Discussion 就支持 RSS 订阅了。

比如 https://github.com/just-talks/tech-talks/discussions 对应的 RSS 地址就是 https://github.com/just-talks/tech-talks/discussions.atom

下图是我用 Feedly 订阅的界面。

![CleanShot 2023-08-01 at 21 24 48@2x](https://github.com/just-talks/tech-talks/assets/1998490/5cf3f17f-2006-4275-9916-ced0dacfeaad)

于是我在[版块目录](https://github.com/just-talks/tech-talks#%E7%89%88%E5%9D%97%E7%9B%AE%E5%BD%95)里加上了 RSS 链接，方便各位订阅。

**Comment by shenweiyan** [comment]
这个挺好~
